bedrock/firefox/tests/test_views.py

```diff
@@ -634,6 +634,37 @@ def test_firefox_home(self, render_mock):
         assert template == ["firefox/home/index-master.html"]
 
 
+class TestFirefoxGA(TestCase):
+    def assert_ga_attr(self, response):
+        doc = pq(response.content)
+        links = doc(".mzp-c-button")
+        for link in links.items():
+            cta_data = link.attr("data-cta-type")
+            cta_link = link.attr("data-link-type")
+            if cta_data:
+                contain_cta = any(cta_data in item for item in ["link", "button"])
```

Comment:
Nit/suggestion: `contain_cta` implies (to me) a plural and leaves me uncertain what the type might be, wheras `contains_cta` is more clearly a boolean to me.